[DOCS-5691] update jenkins parameters (#5215)
jenkins-infra/jenkins.io: content/doc/book/installing/_jenkins-command-parameters.adoc

==== Java 11, Java 8u252, and later

> IMPORTANT: Starting with weekly release line *2.353* Java 11 will be required.

Reviewer: ```suggestion
IMPORTANT: Starting with weekly release line *2.353*, Java 11 will be required.
```

> The Long Term Support release line will not require Java 11 until September, at which point Java 8 support will be depreciated.

Reviewer: ```suggestion
The Long Term Support release line will not require Java 11 until September, at which point Java 8 support will be deprecated.
```

> Java 11, Java 8 update 252 and Java 8 versions after update 252 can run the ALPN TLS extension by installing the Jetty ALPN java server jar and passing it as a java command line argument.

Reviewer: ```suggestion
Java 11 and Java 8 versions later than update 252 can run the ALPN TLS extension by installing the Jetty ALPN Java server jar and passing it as a Java command line argument.
```
